From 5cb48760d1f0724a89967984a6220eb83a60ad78 Mon Sep 17 00:00:00 2001
From: "rongjun.chen" <rongjun.chen@amlogic.com>
Date: Thu, 6 May 2021 14:49:20 +0800
Subject: [PATCH] wifi: fix wifi not connect no internet ap [1/1]

PD#SWPL-48755

Problem:
1/wifi can not auto connect no internet ap

Solution:
not set disconnect for not internet ap

Verify:
s905y4

Change-Id: Ib7e6f47c3f1cead2efc6420eab3e09efc7a7f65b
---
 services/core/java/com/android/server/ConnectivityService.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/ConnectivityService.java b/services/core/java/com/android/server/ConnectivityService.java
index 77cd5d2ffdab..7f58af41a8f9 100644
--- a/services/core/java/com/android/server/ConnectivityService.java
+++ b/services/core/java/com/android/server/ConnectivityService.java
@@ -4038,7 +4038,7 @@ public class ConnectivityService extends IConnectivityManager.Stub
         // to a network that provides no or limited connectivity is not useful, because the user
         // cannot use that network except through the notification shown by this method, and the
         // notification is only shown if the network is explicitly selected by the user.
-        nai.asyncChannel.sendMessage(NetworkAgent.CMD_PREVENT_AUTOMATIC_RECONNECT);
+//        nai.asyncChannel.sendMessage(NetworkAgent.CMD_PREVENT_AUTOMATIC_RECONNECT);
 
         // TODO: Evaluate if it's needed to wait 8 seconds for triggering notification when
         // NetworkMonitor detects the network is partial connectivity. Need to change the design to
-- 
2.29.0

